java/client/test/org/openqa/selenium/ElementAttributeTest.java

```diff
@@ -227,6 +228,14 @@ public void testShouldReturnTheContentsOfATextAreaAsItsValue() {
   }
 
   @Test
+  public void testShouldReturnInnerHtml() {
+    driver.get(pages.simpleTestPage);
+
+    String html = driver.findElement(By.id("wrappingtext")).getAttribute("innerHTML");
```

Comment:
I wonder i'll never need to test any testContent attribute ! it seems there are some magic keys in WebDriver... Any reason the API was not extended to support methods like getHtml or getNodeText ?